### PR TITLE
Register pi's domain (pi19044.is-a.dev)

### PR DESCRIPTION
**Pull request by user0-07161**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website.  >> Content is there tho
- [X] The website is reachable.  
- [X] You're not using Vercel.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://pi19044.github.io/

### CODE REVIEW

**Comment by pi19044** [comment]
im waiting...

File: domains/pi19044.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "PrelevatedInsider19044's website",
+    "repo": "https://github.com/pi19044/pi19044.github.io",
+    "owner": {
+    	"username": "pi19044",
+    	"email": "membermadde.insiddev25120@hotmail.com"
+    },
+
+    "record": {
+    	"CNAME": "pi19044.is-a.dev"
```
Comment (CuteDog5695) [inline domains/pi19044.json:10]: Uh you can't put CNAME record as the one you are trying to get, maybe you wanted like this?
```suggestion
    	"CNAME": "pi19044.github.io"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> im waiting...

There exists a thing called patience :)

**Comment by user0-07161** [comment]
fixed :)

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**